alica_engine/include/engine/logging/LoggingUtil.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "engine/logging/AlicaLogger.h"
+namespace alica
+{
+namespace Logging
```

Comment:
namespace `Logging` is not required since the logging methods are anyway accessed via the `LoggingUtil` class. Prevents having to type `Logging::LoggingUtil`. Maybe also rename `LoggingUtil` to `Logging`